docker/indexer.Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM python:3.9
+
+# Install dependencies
+RUN apt-get update
+RUN apt-get install libpq-dev postgresql-client -y
+
+# Set working directory
+WORKDIR /home/indexer
+
+# Copy source code
+COPY . .
+
+# Install python dependencies
+RUN pip install poetry
+RUN pip install load_dotenv
+RUN pip install psycopg2
+RUN pip install sentry_sdk
+RUN pip install tenacity
```

Comment:
Lines 15-18 shouldn't be necessary, poetry install should pick these up since we included them as dependencies via poetry as a package manager

Reply:
i.e. https://github.com/regen-network/indexer/blob/main/poetry.lock#L258

poetry.lock and poetry install are basically like package.lock and npm install in the python world

Reply:
Thanks! Yea, for some reason I was hitting errors that these were not installed but that was awhile back and seems to be working fine without these lines now. Not sure why that was an issue previously.

Reply:
Ah! Nvm, still hitting the same errors when starting the containers:

```
ModuleNotFoundError: No module named 'dotenv'
```

Reply:
I'm going to leave them for now but maybe there is something we can do to prevent needing them in the future.

Reply:
huh, that's weird